src/erlando.app.src

```diff
@@ -1,6 +1,17 @@
 {application, erlando,
  [{description, "Syntax extensions for Erlang"},
-  {vsn, "%%VSN%%"},
+  {vsn, git},
   {modules, []},
   {registered, []},
-  {applications, []}]}.
+  {applications, []},
+  {pkg_name, "rabbitmq_erlando"},
+  {maintainers, [
+    "VMware, Inc"
```

Comment:
This is not correct. VMware or Pivotal do not maintain Erlang, or even use it (as far as I know).